libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -553,34 +551,30 @@ internal class MapboxTripSession(
     }
 
     private fun checkBannerInstructionEvent(
-        progress: RouteProgress,
         action: (BannerInstructions) -> Unit
     ) {
-        if (bannerInstructionEvent.isOccurring(progress)) {
-            ifNonNull(bannerInstructionEvent.bannerInstructions) { bannerInstructions ->
-                val bannerView = bannerInstructions.view()
-                val bannerComponents = bannerView?.components()
-                ifNonNull(bannerComponents) { components ->
-                    components.forEachIndexed { index, component ->
-                        component.takeIf { it.type() == BannerComponents.GUIDANCE_VIEW }?.let { c ->
-                            components[index] =
-                                c.toBuilder()
-                                    .imageUrl(c.imageUrl()?.plus("&access_token=$accessToken"))
-                                    .build()
-                        }
+        ifNonNull(bannerInstructionEvent.bannerInstructions) { bannerInstructions ->
```

Comment:
Supposedly this should be populated (non-null) from NN only when updated but if you stand still (stale location) this is being sent multiple times. Can be easily tested in `MapboxNavigationActivity`. It seems a regression we introduced at some point. Could you take a look at it @mskurydin @SiarheiFedartsou when you have a chance? 🙏